wdl2cwl/tests/test_miniwdl.py

```diff
@@ -15,7 +15,7 @@ def get_file(path: str) -> str:
 @pytest.mark.parametrize(
     "wdl_path,cwl_path",
     [
-        # ("wdl_files/bowtie_1.wdl", "oop_cwl_files/bowtie_1.cwl"),
+        ("wdl_files/bowtie_1.wdl", "oop_cwl_files/bowtie_1.cwl"),
```

Comment:
? I thought this PR is about bcftools stats?

Reply:
This was to pass the bowtie_1 that was already in the test. Both bowtie_1 and bcftools_stats had the sep function which is the equivalent of the join function, fixing the function for bcftools_stats caused error for bowtie_1.  I later discovered that although the join separator was used for arrays it mattered the content of the items of the array.